ci/run_playbooks.yml

```diff
@@ -56,18 +62,20 @@
           tags:
             - setup
 
-    - name: "Run a playbook locally on the target host"
-      ansible.builtin.shell:
-        cmd: |
-          ANSIBLE_CONFIG=ci/ansible.cfg ansible-playbook -v -e "cifmw_openshift_kubeconfig={{ cifmw_openshift_kubeconfig }}" -e "cifmw_path={{ cifmw_path }}" ci/telemetry_autoscaling.yml
-        chdir: "{{ fvt_dir }}"
-      register: output
+    - name: Run telemetry autoscaling test
+      block:
```

Comment:
What is the reason for adding this to a block?
I don't see an obvious reason for it.

Reply:
Just for Grouping the Related Tasks for running the autoscaling playbook.